src/hermes/commands/deposit/invenio.py

```diff
@@ -17,648 +17,613 @@
 import requests
 
 from hermes import config
+from hermes.commands.deposit.base import BaseDepositPlugin
 from hermes.commands.deposit.error import DepositionUnauthorizedError
 from hermes.error import MisconfigurationError
 from hermes.model.context import CodeMetaContext
 from hermes.model.path import ContextPath
 from hermes.utils import hermes_user_agent
```

Comment:
**nitpick:** This constant looks a lot like a variable...

Reply:
... but looking into the code, it could be just enough of dynamic involved to regard it as a variable.